.gitignore

```diff
@@ -21,3 +21,4 @@ master.zip
 main.zip
 plot/*.LOG
 plot/ocrd*
+.idea/
```

Comment:
Personally I prefer a minimalistic `.gitignore` which only contains entries related to the project.

Ignoring files from individual editors and development tools should be ignores in centralized personal settings.

Reply:
I have removed the offending entry. In fact I have not heard of global *.gitignore* configurations before, as my projects usually have a different directory structure.